rust/kernel/lib.rs

```diff
@@ -22,7 +22,8 @@
     const_raw_ptr_deref,
     const_unreachable_unchecked,
     receiver_trait,
-    try_reserve
+    try_reserve,
+    doc_cfg
```

Comment:
I think you could add a trailing comma to ensure later changes here would only add a line, not change a previously written line (cleaner diff):
```suggestion
    doc_cfg,
```

Reply:
I agree it would be better, but sadly `rustfmt` (in its default config) does not follow it. Instead, we may separate them into individual attributes.

Reply:
Moved to individual `feature`s at https://github.com/Rust-for-Linux/linux/pull/636.